test/bats/test.bats

```diff
@@ -4,6 +4,22 @@ load helpers
 BATS_TESTS_DIR=test/bats/tests
 WAIT_TIME=120
 SLEEP_TIME=1
+WAIT_TIME_DEPLOYMENTS=600
+SLEEP_TIME_DEPLOYMENTS=10
+
+@test "azure-arc deployments have succeeded" {
+    run wait_for_process $WAIT_TIME_DEPLOYMENTS $SLEEP_TIME_DEPLOYMENTS "kubectl wait --for=condition=available deployment --all --namespace azure-arc"
+    assert_success
+}
+
+@test "arc-osm-system deployments have succeeded" {
+    run wait_for_process $WAIT_TIME_DEPLOYMENTS $SLEEP_TIME_DEPLOYMENTS "kubectl wait --for=condition=available deployment --all --namespace arc-osm-system"
+    assert_success
+}
+
+@test "chart version on cluster matches checkout tag" {
+    [[ "$(helm ls -o json --namespace arc-osm-system | jq -r '.[].chart')" != "osm-arc-$(image.tag)" ]]
```

Comment:
Should this be `==` instead of `!=` because the test is checking if chart version matches?